esmf_regrid/tests/unit/experimental/unstructured_scheme/test__regrid_unstructured_to_rectilinear__prepare.py

```diff
@@ -0,0 +1,65 @@
+"""Unit tests for :func:`esmf_regrid.experimental.unstructured_scheme._regrid_unstructured_to_rectilinear__prepare`."""
+
+from iris.coords import AuxCoord
+from iris.cube import Cube
+import numpy as np
+
+from esmf_regrid.esmf_regridder import GridInfo
+from esmf_regrid.experimental.unstructured_regrid import MeshInfo
+from esmf_regrid.experimental.unstructured_scheme import (
+    _regrid_unstructured_to_rectilinear__prepare,
+)
+from esmf_regrid.tests.unit.experimental.unstructured_scheme.test__cube_to_GridInfo import (
+    _grid_cube,
+)
+from esmf_regrid.tests.unit.experimental.unstructured_scheme.test__mesh_to_MeshInfo import (
+    _example_mesh,
+)
+
+
+def _full_mesh():
+    mesh = _example_mesh()
+
+    # In order to add a mesh to a cube, face locations must be added.
+    # These are not used in calculations and are here given a value of zero.
+    mesh_length = mesh.connectivity(contains_face=True).shape[0]
+    dummy_face_lon = AuxCoord(np.zeros(mesh_length), standard_name="longitude")
+    dummy_face_lat = AuxCoord(np.zeros(mesh_length), standard_name="latitude")
+    mesh.add_coords(face_x=dummy_face_lon, face_y=dummy_face_lat)
+    mesh.long_name = "example mesh"
+    return mesh
+
+
+def _flat_mesh_cube():
```

Comment:
Can you document what the object that this returns looks like? i.e. data is all ones?